src/scheduling/cost_models/simulated_quincy_cost_model.h

```diff
@@ -0,0 +1,116 @@
+// The Firmament project
+// Copyright (c) 2014 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
+// Copyright (c) 2015 Ionel Gog <ionel.gog@cl.cam.ac.uk>
+//
+// Quincy scheduling cost model, as described in the SOSP 2009 paper.
+
+#ifndef FIRMAMENT_SCHEDULING_SIMULATED_QUINCY_COST_MODEL_H
+#define FIRMAMENT_SCHEDULING_SIMULATED_QUINCY_COST_MODEL_H
+
+#include <map>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "base/common.h"
+#include "base/types.h"
+#include "misc/utils.h"
+#include "scheduling/common.h"
+#include "scheduling/cost_models/google_runtime_distribution.h"
+#include "scheduling/cost_models/simulated_dfs.h"
+#include "scheduling/cost_models/flow_scheduling_cost_model_interface.h"
+#include "scheduling/knowledge_base.h"
+
+namespace firmament {
+
+typedef int64_t Cost_t;
```

Comment:
We currently define this in every cost model, but this is clearly silly. I will move it into a common header.